Cargo.toml

```diff
@@ -76,8 +76,8 @@ fuel-core-types = { version = "0.23.0", path = "./crates/types", default-feature
 fuel-core-tests = { version = "0.0.0", path = "./tests" }
 fuel-core-xtask = { version = "0.0.0", path = "./xtask" }
 
-# Fuel dependencies
-fuel-vm-private = { version = "0.47.1", package = "fuel-vm", default-features = false }
+# Fuel dependencies 
+fuel-vm-private = { package = "fuel-vm", version = "0.47.1", git = "https://github.com/FuelLabs/fuel-vm", branch = "feature/regenesis_off_chain", default-features = false }
```

Comment:
Why do we need to use `master` of `fuel-vm`? What doesn't work with the `0.47` release?=)

Reply:
The `fuel-vm` branch `feature/regenesis_off_chain` was forked off of the `0.47.1` tag. It includes `serde::Serialize` and `serde::Deserialize` implementations for `double_key` needed for some tables.

The issue is that I cannot merge that because that would require a new release but there seems to be some changes after `0.47.1` that aren't compatible with `fuel-core`'s `master`. 

Any ideas?

Reply:
Could you create PR on the `fuel-vm` side with se/de please?=) I think we will release it this week

Reply:
[PR](https://github.com/FuelLabs/fuel-vm/pull/704)